system/lib/compiler-rt/lib/sanitizer_common/sanitizer_emscripten.cpp

```diff
@@ -127,6 +127,9 @@ u64 MonotonicNanoTime() {
   return (u64)ts.tv_sec * (1000ULL * 1000 * 1000) + ts.tv_nsec;
 }
 
+// FIXME implement on this platform.
+void GetMemoryProfile(fill_profile_f cb, uptr *stats) {}
```

Comment:
How does this help get a test to pass? Is this wasm64-specific?

Reply:
Reverted